apps/editor/index.d.ts

```diff
@@ -740,6 +740,8 @@ declare namespace toastui {
 
     constructor(options: EditorOptions);
 
+    public eventManager: EventManager;
```

Comment:
It's not original purpose to provide this property as a public. What do you need it for?

Reply:
I need it to listen for button clicks in the toolbar:
```js
    const toolbar = editor.getUI().getToolbar()
    toolbar.insertItem(toolbar.getItems().length, {
      type: 'button',
      options: {
        className: 'tui-image',
        event: 'upload-image',
        tooltip: 'Insert image',
        text: '',
        style: '',
      },
    })
    editor.eventManager.addEventType('upload-image')
   ```

Reply:
I got it. Thanks!